contributed_definitions/NXcoordinate_system_set.nxdl.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<definition category="base" extends="NXobject" name="NXcoordinate_system_set" type="group" xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
change

```xml
    xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance"
```

to

```xml
    xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
```

Reply:
Check all nxdl.xml files in this PR (#996), all #995, #997, and #998.  Revise each as needed.

Reply:
All other issues are fixed in this and the other PRs, this issue will also be fixed tomorrow.

Reply:
this and all the other related PRs are now updated and ready for being merged. #995 #996 #997 #998.